units/meson.build

```diff
@@ -257,7 +258,7 @@ in_units = [
         ['systemd-vconsole-setup.service',       'ENABLE_VCONSOLE'],
         ['systemd-volatile-root.service',        'ENABLE_INITRD'],
         ['systemd-repart.service',               'ENABLE_REPART',
-         'sysinit.target.wants/ initrd-root-fs.target.wants/'],
+         'root-fs-ready.target.wants/ initrd-root-fs.target.wants/'],
```

Comment:
so, is "root-fs-ready.target" supposed to be a "passive" unit or an "active" one? i.e. shall the unit that needs the root fs pull ready it in or the unit that readies the root fs?

I think it should be the latter, as in many scenarios the root fs is ready right away, and we could do without the synchronization point then. But that suggests that systemd-growfs-root.service + systemd-repart.service + systemd-fsck-root.service should pull in the unit via Wants= and order themselves before that. But it shouldn't be root-fs-ready.target that pulls either of them in.

Reply:
you documented the new unit among "passive" units in the man page, btw, which sounds good to me.

Reply:
OK, now I'm fully confused. In the previous round, you said that it should be active. I remade the unit to be active following that review.

Reply:
even if i said different things before I think it should be passive, i.e. pulled in by systemd-fsck-root.service, systemd-growfs-root.service and the lot.